pilot/docker/Dockerfile.proxyv2

```diff
@@ -14,12 +14,18 @@ COPY gcp_envoy_bootstrap.json /var/lib/istio/envoy/gcp_envoy_bootstrap_tmpl.json
 RUN chown -R istio-proxy /var/lib/istio
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
-FROM gcr.io/distroless/cc@sha256:f81e5db8287d66b012d874a6f7fea8da5b96d9cc509aa5a9b5d095a604d4bca1 as distroless
+FROM gcr.io/distroless/cc-debian10@sha256:5bc7ef365a81ac2a58de8b9606d6840ea1aa4e03847137671671384b0b1700d4 as distroless-amd64
+COPY --from=default /usr/lib/x86_64-linux-gnu/xtables/ /usr/lib/x86_64-linux-gnu/xtables
+COPY --from=default /usr/lib/x86_64-linux-gnu/ /usr/lib/x86_64-linux-gnu
+
+FROM gcr.io/distroless/cc-debian10@sha256:5bc7ef365a81ac2a58de8b9606d6840ea1aa4e03847137671671384b0b1700d4 as distroless-arm64
+COPY --from=default /usr/lib/aarch64-linux-gnu/xtables/ /usr/lib/aarch64-linux-gnu/xtables
+COPY --from=default /usr/lib/aarch64-linux-gnu/ /usr/lib/aarch64-linux-gnu
+
+FROM distroless-${TARGETARCH:-amd64} as distroless
```

Comment:
default value here to support `buildx` and `build` both.

Reply:
One question: let's say that the argument `--platform=linux/amd64` is passed to the `docker buildx`.
Doesn't the code block from the line 21 to 23 break the build? Or it is not a problem with the multi-arch image?

Reply:
Using `docker buildx`, stage `distroless-arm64` will skip when `--platform=linux/amd64` only.